src/gdrdrv/gdrdrv.h

```diff
@@ -23,9 +23,13 @@
 #ifndef __GDR_DRV_H__
 #define __GDR_DRV_H__
 
+#define _GDRDRV_STRINGIFY(s)           #s
+#define _GDRDRV_TOSTRING(s)            _GDRDRV_STRINGIFY(s)
```

Comment:
usually identifiers starting with underscore are considered "reserved".
can you get rif of it?